express/components/common.py

```diff
@@ -192,6 +192,16 @@ class ExpressDatasetHandler(ABC, Generic[IndexT, DataT]):
      Pandas DataFrame or a Spark RDD.
     """
 
+    @property
+    @abstractmethod
+    def data_sources_in(self):
```

Comment:
I would expect these attributes on a class which represents the component specification.